README.md

```diff
@@ -28,22 +13,47 @@ The analytics library contains components and modules that are used in DHIS 2 an
 -   [dhis2/data-visualizer-app](https://github.com/dhis2/data-visualizer-app)
 -   [dhis2/maps-app](https://github.com/dhis2/maps-app)
+-   [dhis2/line-listing-app](https://github.com/dhis2/line-listing-app)
 
 [Module layout documentation](./docs/module-layout.md)
 
 ## Publishing
 
 The analytics package is published to npm as @dhis2/analytics.
 
-To publish, simply mark the commit using [semantic release terminology](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) (see notes below for branch-specific restrictions). Once committed, github actions will take care of publishing the new version to npm.
+To publish, mark the commit using [semantic release terminology](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) (see notes below for branch-specific restrictions). Once committed, github actions will take care of publishing the new version to npm.
 
 ### master branch
 
 The master branch follows semantic versioning according to spec.
 
 ### .x branches
 
-Commits to .x branches (e.g. 16.x) cannot trigger a major version bump, even if it is technically a breaking change. This is because the next version has already been published. Additionally, branches that use .x for the patch version (e.g. 11.0.x, 2.4.x), cannot trigger a minor version bump. In the unlikely case that you need to commit a change that would trigger a version bump that's not possible, you will have to mark it to only trigger a patch or minor bump respectively, then make sure to update the apps that are locked to the .x version of analytics
+Commits to .x branches (e.g. 24.x) cannot trigger a major version bump, even if it is technically a breaking change. This is because the next version has already been published. Additionally, branches that use .x for the patch version (e.g. 11.0.x, 2.4.x), cannot trigger a minor version bump. In the unlikely case that you need to commit a change that would trigger a version bump that's not possible, you will have to mark it to only trigger a patch or minor bump respectively, then make sure to update the apps that are locked to the .x version of analytics
+
+## Publishing pre-release versions during app development
+
+Builds for all non-production branches are automatically copied to [d2-ci/analytics](https://github.com/d2-ci/analytics) for use during development and testing, prior to production release.
+
+To test changes in a development branch, change the analytics dependency of package.json of the app you are testing with. There are a few options:
+
+1. point to a specific commit:
+
+```
+"dependencies": {
+        "@dhis2/analytics": "git+https://github.com/d2-ci/analytics.git#70249ebe8be39051fa10142f850de449e1ec488c",
+        ...
+}
+```
+
+2. point to a branch:
+
+```
+"dependencies": {
+        "@dhis2/analytics": "git+https://github.com/d2-ci/analytics.git#chore/some-chore",
+        ...
+}
+```
```

Comment:
OK, this illustrates that what I suggested in my previous commit is already implemented.